org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
```

Comment:
Maybe bump it to 5.15?

Reply:
Yes

Reply:
Build fine with runtime-version": "5.15" here see:
https://qelectrotech.org/forum/viewtopic.php?pid=14448#p14448